### DISCUSSION TITLE
Maybe more explanation on how to improve Google LightHouse score?

### DISCUSSION THREAD

**Discussion by ebiggerr** [OysterD3/frontend-practice#2] [Ideas]
It mentioned that the dev should work on improving the score but how?

Some ideas:
1. Suitable image size/resolution 
2. Take `webpack` as the example, do know that there are differences between `development build` and `production build`.
3. Purge unused CSS
4. Use minified asset files (such as CSS and JS)

From,
Not an expert too 😄

**Comment by OysterD3** [comment]
Great idea! Can you create a PR and amend README.md?

**Reply by ebiggerr** [reply]
Sure. Probably I will add some resource on the OPEN API too. But do you have a template or format in your mind? In the long run, fitting all information in one MD file seems to be not friendly in terms of readability.

#3

**Reply by OysterD3** [reply]
I will leave it first, will find another time to polish
